### PR TITLE
Backport print v3 template from demo

### LINKED ISSUE #1351: Wrong template name specified
https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/create/vars_%2Bpackage%2B.yaml_tmpl#L45-L46

### PR DESCRIPTION
**Pull request by sbrunner**
Also fix  #1351

### CODE REVIEW

File: c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako [hunk 1]
```diff
@@ -3,25 +3,30 @@ throwErrorOnExtraParameters: true
 templates:
     A4 portrait: !template
         reportTemplate: A4_Portrait.jrxml
-        tableData: jrDataSource
         attributes:
-            title: !string {}
-            comments: !string {}
+            title: !string
+                default: ""
```
Comment (kalbermattenm) [inline c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako]: Setting these kind of `defaults` values makes Tomcat raise an error...:

Are you sure that this is correct ?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako]: ... yes, what kind of error do you have ?

Comment (kalbermattenm) [inline c2cgeoportal/scaffolds/create/print/print-apps/+package+/config.yaml.mako]: Just by adding `default` under `title`like this:

```
      title: !string {}
        default: ""
```
I get the following error:

```
2015-05-19 13:25:24,283 ERROR [c2cgeoportal.views.proxy][Dummy-4] <html><head><title>Apache Tomcat/7.0.21 - Rapport d''e
rreur</title><style><!--H1 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:22px;} H2
 {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;font-size:16px;} H3 {font-family:Tahoma,Arial
,sans-serif;color:white;background-color:#525D76;font-size:14px;} BODY {font-family:Tahoma,Arial,sans-serif;color:black;
background-color:white;} B {font-family:Tahoma,Arial,sans-serif;color:white;background-color:#525D76;} P {font-family:Ta
homa,Arial,sans-serif;background:white;color:black;font-size:12px;}A {color : black;}A.name {color : black;}HR {color :
#525D76;}--></style> </head><body><h1>Etat HTTP 500 - </h1><HR size="1" noshade="noshade"><p><b>type</b> Rapport d''exce
ption</p><p><b>message</b> <u></u></p><p><b>description</b> <u>Le serveur a rencontr├® une erreur interne () qui l'a emp
├¬ch├® de satisfaire la requ├¬te.</u></p><p><b>exception</b> <pre>org.springframework.web.util.NestedServletException: R
equest processing failed; nested exception is java.lang.RuntimeException: Error occurred while reading configuration fil
e 'file:/C:/ApacheTomcat7/webapps/print-michael/print-apps/sitn/config.yaml':
        org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:965)
        org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:844)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:621)
        org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:829)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
        com.codahale.metrics.servlet.AbstractInstrumentedFilter.doFilter(AbstractInstrumentedFilter.java:97)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.jav
a:118)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.j
ava:84)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113)

        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFi
lter.java:113)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHold
erAwareRequestFilter.java:154)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter
.java:150)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyn
cManagerIntegrationFilter.java:50)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFil
ter.java:87)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.channel.ChannelProcessingFilter.doFilter(ChannelProcessingFilter.java:14
4)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192)
        org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
        org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:343)
        org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:260)
        org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.mapfish.print.servlet.RequestSizeFilter.doFilter(RequestSizeFilter.java:59)
</pre></p><p><b>cause m├¿re</b> <pre>java.lang.RuntimeException: Error occurred while reading configuration file 'file:/
C:/ApacheTomcat7/webapps/print-michael/print-apps/sitn/config.yaml':
        org.mapfish.print.servlet.ServletMapPrinterFactory.create(ServletMapPrinterFactory.java:139)
        org.mapfish.print.servlet.MapPrinterServlet.getCapabilities(MapPrinterServlet.java:582)
        sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        java.lang.reflect.Method.invoke(Unknown Source)
        org.springframework.web.bind.annotation.support.HandlerMethodInvoker.invokeHandlerMethod(HandlerMethodInvoker.ja
va:176)
        org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.invokeHandlerMethod(AnnotationMeth
odHandlerAdapter.java:440)
        org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.handle(AnnotationMethodHandlerAdap
ter.java:428)
        org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:925)
        org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:856)
        org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:953)
        org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:844)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:621)
        org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:829)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
        com.codahale.metrics.servlet.AbstractInstrumentedFilter.doFilter(AbstractInstrumentedFilter.java:97)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.jav
a:118)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.j
ava:84)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113)

        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFi
lter.java:113)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHold
erAwareRequestFilter.java:154)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter
.java:150)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyn
cManagerIntegrationFilter.java:50)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFil
ter.java:87)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.channel.ChannelProcessingFilter.doFilter(ChannelProcessingFilter.java:14
4)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192)
        org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
        org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:343)
        org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:260)
        org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.mapfish.print.servlet.RequestSizeFilter.doFilter(RequestSizeFilter.java:59)
</pre></p><p><b>cause m├¿re</b> <pre>while parsing a block mapping
 in 'reader', line 13, column 7:
          title: !string {}
          ^
expected &lt;block end&gt;, but found BlockMappingStart
 in 'reader', line 14, column 9:
            default: &quot;&quot;
            ^

        org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:570)
        org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:158)
        org.yaml.snakeyaml.parser.ParserImpl.checkEvent(ParserImpl.java:143)
        org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:226)
        org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:155)
        org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:233)
        org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:155)
        org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:233)
        org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:155)
        org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:233)
        org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:155)
        org.yaml.snakeyaml.composer.Composer.composeDocument(Composer.java:122)
        org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:105)
        org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:120)
        org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:481)
        org.yaml.snakeyaml.Yaml.load(Yaml.java:424)
        org.mapfish.print.config.ConfigurationFactory.getConfig(ConfigurationFactory.java:86)
        org.mapfish.print.MapPrinter.setConfiguration(MapPrinter.java:80)
        org.mapfish.print.servlet.ServletMapPrinterFactory.create(ServletMapPrinterFactory.java:128)
        org.mapfish.print.servlet.MapPrinterServlet.getCapabilities(MapPrinterServlet.java:582)
        sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        java.lang.reflect.Method.invoke(Unknown Source)
        org.springframework.web.bind.annotation.support.HandlerMethodInvoker.invokeHandlerMethod(HandlerMethodInvoker.ja
va:176)
        org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.invokeHandlerMethod(AnnotationMeth
odHandlerAdapter.java:440)
        org.springframework.web.servlet.mvc.annotation.AnnotationMethodHandlerAdapter.handle(AnnotationMethodHandlerAdap
ter.java:428)
        org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:925)
        org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:856)
        org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:953)
        org.springframework.web.servlet.FrameworkServlet.doGet(FrameworkServlet.java:844)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:621)
        org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:829)
        javax.servlet.http.HttpServlet.service(HttpServlet.java:722)
        com.codahale.metrics.servlet.AbstractInstrumentedFilter.doFilter(AbstractInstrumentedFilter.java:97)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:330)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.jav
a:118)
        org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.j
ava:84)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:113)

        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:103)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFi
lter.java:113)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHold
erAwareRequestFilter.java:154)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:45)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.authentication.www.BasicAuthenticationFilter.doFilter(BasicAuthenticationFilter
.java:150)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyn
cManagerIntegrationFilter.java:50)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFil
ter.java:87)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.access.channel.ChannelProcessingFilter.doFilter(ChannelProcessingFilter.java:14
4)
        org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:342)
        org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:192)
        org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:160)
        org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:343)
        org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:260)
        org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:88)
        org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107)
        org.mapfish.print.servlet.RequestSizeFilter.doFilter(RequestSizeFilter.java:59)
</pre></p><p><b>note</b> <u>La trace compl├¿te de la cause m├¿re de cette erreur est disponible dans les fichiers journa
ux de Apache Tomcat/7.0.21.</u></p><HR size="1" noshade="noshade"><h3>Apache Tomcat/7.0.21</h3></body></html>
```

**Comment by sbrunner** [comment]
In our file we don't have:
```
      title: !string {}
        default: ""
```
but:
```
      title: !string
        default: ""
```
(without `{}`) the first isn't valid YAML...

**Comment by kalbermattenm** [comment]
-1 for me...

Thanks @sbrunner....

**Pull request merged**